index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  	<!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <link rel="stylesheet" type="text/css" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>FORMS</title>
+  </head>
+  <body>
+
+
+	<div class="container-fluid">
+		<div class="row py-2">
+			<div class="col-8"></div>
+			<div class="col-4"><span class="smallFont float-right">Already have an Intuit account?<br>
+              <a href="#">I forgot my User ID or Password</a></span>
+			<span class="float-right"><button class="btn btn-default btn-blue smallFont"><i class="fa fa-lock" ></i> Sign In</button></span>
+		</div>
+		</div>
+
+		<div class="row text-center imgSide my-3">
+			<div class="col">
+			<p class="text-center"><img class="img-fluid" src="img/logo.png"/ alt="website logo"></p>
+			<p class="text-center"><img class="img-fluid" src="img/mint.png"/ alt="mint logo"><img class="img-fluid" src="img/quickbooks.png" alt="quickbooks logo" /><img class="img-fluid" src="img/turbotax.png" alt="turbotax logo" /></p>
+
+			</div>
+		</div>
+
+		<div class="row">
+			<div class="col">
+			<div class="white-box">
+              <div class="formtitlearea">
+				<h4 class="text-center">Create an Intuit account</h4>
+				<p class="text-center formsubtitle">One account for everything Intuit, including Mint. <a href="#">Learn more</a></p>
+				</div>
+        <div class="formarea">
+          <form action="myform.cgi">
+          	<div class="form-group">
+			    <label for="emailaddress">Email address</label>
+			    <input type="email" class="form-control" id="emailaddress" name="email">
```

Comment:
`AWESOME`

Good job associating the `<label>` with an `<input>` element. :clap: :+1: 

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```


You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)